lib/ActivityStream.jsm

```diff
@@ -145,7 +145,7 @@ const PREFS_CONFIG = new Map([
   }],
   ["section.highlights.rows", {
     title: "Number of rows of Highlights to display",
-    value: 2,
+    value: 1,
```

Comment:
why this change?

Reply:
for a single row:
![Screen Shot 2019-06-14 at 7 04 51 PM](https://user-images.githubusercontent.com/438537/59545722-654a6d00-8ed7-11e9-963c-d307e7ed1773.png)